paasta_tools/cli/cmds/check.py

```diff
@@ -308,6 +309,7 @@ def paasta_check(args):
     marathon_deployments_check(service_path)
     sensu_check(service, service_path)
     smartstack_check(service, service_path)
+    paasta_validate(None, service_path)
```

Comment:
I'm always confused when I see this kind of "None" thing in an arg. None what?
Can you kwargs this?